test/Validation.test.js

```diff
@@ -146,17 +166,20 @@ describe("Validation", function() {
 		]
 	}];
 	testCases.forEach(function(testCase) {
-		it("should fail validation for " + testCase.name, function() {
-			try {
-				webpack(testCase.config);
-			} catch(e) {
-				if(!(e instanceof WebpackOptionsValidationError))
-					throw e;
-				e.message.should.startWith("Invalid configuration object.");
-				e.message.split("\n").slice(1).should.be.eql(testCase.message);
-				return;
-			}
-			throw new Error("Validation didn't fail");
-		})
+		if(testCase.name === "empty entry bundle array") {
```

Comment:
@sokra my bad! Thanks for the suggestion, I have removed the leftovers from testing.